setup.py

```diff
@@ -52,7 +52,7 @@
     'bravado>=9.0.6,<10.2',
     'celery>=4.1.0,<4.3',
     'click>=7,<8',
-    'reana-commons>=0.5.0.dev20190213,<0.6.0'
+    'reana-commons[kubernetes]>=0.5.0.dev20190213,<0.6.0'
```

Comment:
Please also increase the version of r-w-e-serial at the same time to the current YYYMMDD.